### PR TITLE
Update to cmake-js major release version 7

### PR DESCRIPTION
**Pull request by bertschneider**
cmake-js 7 suggests a few changes to the build file.

### CODE REVIEW

File: examples/dry_run.sh [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 set -e
 
-for dir in $(ls -d */); do
+for dir in */; do
+  echo "--- Executing example in \"${dir}\" -----------------"
   cd "${dir}";
-  npm install
-  npm run dryRun
-  cd ..
+  if [ -f "package.json" ]; then
```
Comment (kyakdan) [inline examples/dry_run.sh:7]: Let's keep this in sync with [dry_run.bat](https://github.com/CodeIntelligenceTesting/jazzer.js/blob/2a6643fbf73ac00c57a2b22efe989f0c2cba6924/examples/dry_run.bat) and add a similar check there.

File: packages/fuzzer/CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 3.10) # shipped with Ubuntu 18.04
+# Suggested settings by cmake-js
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW)
+cmake_policy(SET CMP0042 NEW)
```
Comment (kyakdan) [inline packages/fuzzer/CMakeLists.txt:4]: Can you add comments on why these additions are necessary?

File: examples/dry_run.sh (hunk 1 above)
Comment (bertschneider) [reply examples/dry_run.sh:7]: Good catch, done!

File: packages/fuzzer/CMakeLists.txt (hunk 2 above)
Comment (bertschneider) [reply packages/fuzzer/CMakeLists.txt:5]: Added the explanation from the cmake-js release log. Is  that enough?

Comment (kyakdan) [reply packages/fuzzer/CMakeLists.txt:5]: These changes affect macOS and Windows respectively. Can we move them down the file where we perform os-dependant settings?

Comment (bertschneider) [reply packages/fuzzer/CMakeLists.txt]: done

**Review by kyakdan — APPROVED** [review]

**Pull request merged**